tools/mpremote/mpremote/main.py

```diff
@@ -197,10 +197,25 @@ def argparse_filesystem():
         None,
         "enable verbose output (defaults to True for all commands except cat)",
     )
+    size_group = cmd_parser.add_mutually_exclusive_group()
+    size_group.add_argument(
+        "--size",
+        "-s",
+        default=False,
+        action="store_true",
+        help="show file size in bytes(tree command only)",
+    )
+    size_group.add_argument(
+        "--human",
```

Comment:
I suggest allowing this to take the short form `-h` as well as the long form, to match unix tree.

Reply:
I tried initially, but that conflicts with the existing `-h/--help` of mpremote fs

I can change that, but that may be a breaking change to some of the wrapper modules although i think the risk is low.
Also de docs state clearly that it should not be done/ depended upon.

Reply:
Oh, right.  The `-h`/`--help` options are added automatically to all the subcommands...

I think it's fair enough to repurpose `-h` for whatever the subcommand needs, in this case it would be shorthand for `--human`.  Then there's always the long form `--help` to get help if the user needs that.